src/frontend/screens/Settings/components/WineVersionSelector.tsx

```diff
@@ -31,6 +31,7 @@ export default function WineVersionSelector() {
       }
       setRefreshing(false)
     }
+    window.api.handleWineVersionsUpdated(getAltWine)
     getAltWine()
```

Comment:
Return the return value of `handleWineVersionsUpdated` to remove the listener once the component is unmounted
```suggestion
    getAltWine()
    return window.api.handleWineVersionsUpdated(getAltWine)
```